src/main/java/org/trigon/hardware/misc/leds/CANdleLEDStrip.java

```diff
@@ -203,7 +202,7 @@ private void setSectionColor(int amountOfSections, int LEDSPerSection, Supplier<
         }
     }
 
-    private boolean isSimulation(Runnable simulationAction) {
+    private boolean runIfInSimulation(Runnable simulationAction) {
```

Comment:
This should either be a getter and return the boolean, or be a setter and run the runnable. Also, consider the name `runInSimulation`.

Reply:
That removes the point of extracting it to a method